luxjs/grid/grid.js

```diff
@@ -327,8 +327,10 @@
                 scope.gridState = gridDefaults.gridState;
                 scope.gridFilters = gridDefaults.gridFilters;
 
+                var reprPath = options.reprPath || $lux.window.location;
+
                 scope.objectUrl = function(entity) {
-                    return $lux.window.location + '/' + entity[scope.gridOptions.metaFields.id];
+                    return reprPath + '/' + entity[scope.gridOptions.metaFields.id];
                 };
 
```

Comment:
I'm not sure it is the best way to handle this.
What about ``objectUrl`` be an optional function in the ``options`` object, rather than ``reprPath``?
Or that would be not easy to handle form the application side?